N3.js/src/main/scala/org/w3/banana/n3js/io/N3jsTurtleParser.scala

```diff
@@ -0,0 +1,45 @@
+package org.w3.banana
+package n3js
```

Comment:
package names does not match naming convention - eg see https://github.com/w3c/banana-rdf/issues/139